ladybug/dt.py

```diff
@@ -79,10 +79,15 @@ def from_moy(cls, moy, leap_year=False):
         Args:
             moy: An integer value 0 <= and < 525600
         """
-        num_of_minutes_until_month = (0, 44640, 84960, 129600, 172800, 217440,
-                                      260640, 305280, 349920, 393120, 437760,
-                                      480960, 525600)
-
+        if not leap_year:
+            num_of_minutes_until_month = (0, 44640, 84960, 129600, 172800, 217440,
+                                          260640, 305280, 349920, 393120, 437760,
+                                          480960, 525600)
+        else:
```

Comment:
I was thinking that this would look cleaner by just declaring 1440 as a variable once but then I realized that would unnecessarily create an extra variable that gets kept in memory.  So this is good.